src/Components/VideoCard/videoCard.jsx

```diff
@@ -1,22 +1,54 @@
 import { useData } from "../../context/data";
-import { AiOutlineLike, AiFillLike } from "react-icons/ai";
+import { MdPlaylistAdd, MdWatchLater } from "react-icons/md";
+import { BsThreeDotsVertical } from "react-icons/bs";
 import "./videoCard.css";
 import { useNavigate } from "react-router-dom";
-export const VideoCard = () => {
-  const { filterData } = useData();
+import { useState } from "react";
+export const VideoCard = ({ prop }) => {
+  const { data } = prop;
+  const [show, setShow] = useState(false);
+  const { dispatch } = useData();
+  const token = localStorage.getItem("token");

```

Comment:
suggestion: get the token in context once and maintain from there instead of getting it on every page